packages/menu/src/MenuItem.ts

```diff
@@ -460,13 +471,21 @@ export class MenuItem extends LikeAnchor(
         super.firstUpdated(changes);
         this.setAttribute('tabindex', '-1');
         this.addEventListener('keydown', this.handleKeydown);
+        this.addEventListener('mouseover', this.handleMouseover);
         this.addEventListener('pointerdown', this.handlePointerdown);
         this.addEventListener('pointerenter', this.closeOverlaysForRoot);
         if (!this.hasAttribute('id')) {
             this.id = `sp-menu-item-${randomID()}`;
         }
     }
 
+    handleMouseover(event: MouseEvent): void {
```

Comment:
Removes focus styling from hovered menuitem. Stlye would return if keyboard moved elsewhere and then came back to the item.